pandas/tests/io/formats/test_to_html.py

```diff
@@ -235,6 +235,22 @@ def test_to_html_truncate(datapath):
     assert result == expected
 
 
+def test_html_formatters_empty():
```

Comment:
Can you parametrize these tests instead? Looks be the same as the one below

Reply:
I joined the two of them in one and parametrized them so that both cases can be tested in the same test. Is this what you meant?